src/WholeBodyControllers/include/WalkingControllers/WholeBodyControllers/QPInverseKinematics.h

```diff
@@ -280,6 +304,21 @@ namespace WalkingControllers
         void setDesiredFeetTwist(const iDynTree::Twist& leftFootTwist,
                                  const iDynTree::Twist& rightFootTwist);
 
+        /**
+         * Set the desired twist of both feet
+         * @param leftHandTwist contain the desired twist of the left hand (MIXED representation);
+         * @param rightHandTwist contain the desired twist of the right hand (MIXED representation).
+         */
+        void setDesiredHandsTwist(const iDynTree::Twist& leftHandTwist,
+                                  const iDynTree::Twist& rightHandTwist);
+
+        /**
+         * Set the desired twist of both feet
+         * @param jointPosition contain the desired joint position used in the retargeting.
+         * @return true/false in case of success/failure.
+         */
+        bool setDesiredRetargetingJoint(const iDynTree::VectorDynSize& jointPosition);
```

Comment:
The documentation seems wrong.

Reply:
Done in c2828ba194b295b916502f36e776a0efe488979e